js/augurs-prophet-js/src/lib.rs

```diff
@@ -176,6 +176,14 @@ impl Prophet {
         self.inner.add_seasonality(name, seasonality.into())?;
         Ok(())
     }
+
+    /// Add a regressor to the model. Name should be one of the column names.
+    /// The extra regressor must be known for both the history and for future dates.
+    #[wasm_bindgen(js_name = "addRegressor")]
+    pub fn add_regressor(&mut self, name: String, regressor: Regressor) -> Result<(), JsError> {
+        self.inner.add_regressor(name, regressor.into());
```

Comment:
We could make regressor optional here too to mimic the Python API where all params except `name` are optional.

```suggestion
    pub fn add_regressor(&mut self, name: String, regressor: Option<Regressor>) -> Result<(), JsError> {
        self.inner.add_regressor(name, regressor.unwrap_or_default().into());
```